pkg/util/docker/docker.go

```diff
@@ -41,7 +41,7 @@ var (
 		CPU:     &CgroupTimesStat{},
 		Memory:  &CgroupMemStat{},
 		IO:      &CgroupIOStat{},
-		Network: &NetworkStat{},
+		Network: new(ContainerNetStats),
```

Comment:
any particular reason for this change? I think it should do the same thing and the other one is shorter ;)

Reply:
A remnant of when that was an alias for `[]*InterfaceNetStats` and the short init didn't work